app/commands/setup.ts

```diff
@@ -20,6 +20,13 @@ export const command = new SlashCommandBuilder()
       .setName("mod-log-channel")
       .setDescription("The channel where moderation reports will be sent")
       .setRequired(true),
+  )
+  .addRoleOption((x) =>
+    x
```

Comment:
What is `x` here? If it's a particular role then i would named it `role`.

Reply:
It's a role option builder, since we have types and it's only used in the one place I'm not overly worried. Autocomplete makes it clear what the available options are more than the name would